CMakeLists.txt

```diff
@@ -630,6 +630,10 @@ if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC" OR "${CMAKE_CXX_SIMULATE_ID}" STREQU
 endif()
 
 if(NOT SWIFT_COMPILER_IS_MSVC_LIKE)
+  # CMake's default for CMAKE_CXX_FLAGS_DBEUG is "-g". Let's add "-O0", because we want to be able
+  # to append CMAKE_CXX_FLAGS_DEBUG to a list of compile flags that already contains a -O flag.
```

Comment:
which list already has `-O`?

Reply:
@mikeash reported this when building a "release swift + debug stdlib" configuration. The stdlib in this case gets the "default compilation flags" and then we append the "debug stdlib" flags.